QUICK_START_GUIDE.md

```diff
@@ -200,17 +211,18 @@
                            min_value: 0.0
    ```
 
-### ote optimize - run optimization of a particular model template
-   This tool optimizes a trained model using NNCF or POT depending on a model format:
-
-   - NNCF optimization used for trained snapshots in framework specific format
-   - POT optimization used for exported model in IR format
+### ote optimize 
+   `ote optimize` optimizes a pre-trained model using NNCF or POT depending on the model format.
+   * NNCF optimization used for trained snapshots in a framework-specific format
+   * POT optimization used for models exported in the OpenVINO IR format
 
```

Comment:
```suggestion
For example:
**Optimise a PyTorch model (.pth) with OpenVINO NNCF:**
```
ote optimize ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --load-weights weights.pth --save-model-to ./nncf_output --save-performance ./nncf_output/performance.json --train-ann-file ./data/car_tree_bug/annotations/instances_default.json --train-data-roots ./data/car_tree_bug/images --val-ann-file ./data/car_tree_bug/annotations/instances_default.json --val-data-roots ./data/car_tree_bug/images
```

**Optimise OpenVINO model (.bin or .xml) with OpenVINO POT:**
```
ote optimize ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --load-weights openvino.xml --save-model-to ./pot_output --save-performance ./pot_output/performance.json --train-ann-file ./data/car_tree_bug/annotations/instances_default.json --train-data-roots ./data/car_tree_bug/images --val-ann-file ./data/car_tree_bug/annotations/instances_default.json --val-data-roots ./data/car_tree_bug/images
```

```